<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSAS2Tab</class>
 <widget class="QWidget" name="GSAS2Tab">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>730</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid grey;border-radius: 10px;margin-top: 1ex; margin-right: 0ex
}
QGroupBox:title {
                           subcontrol-origin: margin;
                           padding: 0 3px;
                           subcontrol-position: top center;
                           padding-top: 0px;
                          padding-bottom: 0px;
                           padding-right: 10px;
                            color: rgb(56, 56, 56)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Load Focused Data</string>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Project Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="project_name_line_edit">
             <property name="text">
              <string>220321script3</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>411</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Focused Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="focused_data_path_line_edit">
             <property name="text">
              <string>Save_gss_305761_307521_bank_1_bgsub.gsa</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>120</y>
            <width>581</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Phase file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="phase_file_path_line_edit">
             <property name="text">
              <string>FE_GAMMA.cif</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_7">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>70</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Data Indexing</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="data_indexing_line_edit">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Refinement Parameters</string>
           </property>
           <widget class="QPushButton" name="refine_button">
            <property name="geometry">
             <rect>
              <x>460</x>
              <y>40</y>
              <width>121</width>
              <height>71</height>
             </rect>
            </property>
            <property name="text">
             <string>Refine in GSAS II</string>
            </property>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>231</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Refinement Method</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="refinement_method_combobox">
               <item>
                <property name="text">
                 <string>Pawley</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Rietveld</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>90</y>
              <width>281</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Override Unit Cell Length</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="override_unitcell_length"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Ã…</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>20</y>
              <width>155</width>
              <height>111</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QCheckBox" name="refine_microstrain_checkbox">
               <property name="text">
                <string>Refine Microstrain</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="refine_sigma_one_checkbox">
               <property name="text">
                <string>Refine Sigma-1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="refine_gamma_y_checkbox">
               <property name="text">
                <string>Refine Gamma (Y)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>4</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Plot</string>
         </property>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>259</y>
            <width>651</width>
            <height>61</height>
           </rect>
          </property>
          <property name="title">
           <string>Fitting Range</string>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>651</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Min </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="min_x_value_plot"/>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>20</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="max_x_value_plot"/>
            </item>
           </layout>
          </widget>
         </widget>
         <zorder>groupBox_4</zorder>
         <zorder></zorder>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
